Gemfile

```diff
@@ -25,13 +27,14 @@ gem 'uglifier', '>= 1.3.0'
 gem 'webpacker'
 
 # Core Samvera
-gem 'active-fedora', '~> 13.2', '>= 13.2.5'
-gem 'active_fedora-datastreams', '~> 0.4'
+#gem 'active-fedora', '~> 14.0'
+gem 'active-fedora', git: "https://github.com/samvera/active_fedora.git", branch: "errors_fix"
+gem 'active_fedora-datastreams', '~> 0.5'
 gem 'hydra-head', '~> 12.0'
-gem 'ldp', '~> 1.0.3'
-gem 'noid-rails', '~> 3.0.1'
+gem 'ldp', '~> 1.1.0'
+gem 'noid-rails', '~> 3.1'
+gem 'om', git: 'https://github.com/avalonmediasystem/om.git', branch: 'ruby3'
```

Comment:
Use a fork because upstream is in `samvera-deprecated`.  We should consider contributing this back up and making a new release of `om` if we can figure out the process.